cmd/infra/powervs/create.go

```diff
@@ -158,13 +160,15 @@ func NewCreateCommand() *cobra.Command {
 	}
 
 	opts := CreateInfraOptions{
+		Name:          "example",
```

Comment:
The cluster name is already a config option in provider-agnostic create cluster layer, we shouldn't redefine it her: https://github.com/openshift/hypershift/blob/753942e8319b4a6f7e259591aef4e53defbdc854/cmd/cluster/cluster.go#L44

Reply:
But this is on infra command, name defined in cluster command won't be accessible here. It is similar to name defined here for AWS infra https://github.com/openshift/hypershift/blob/main/cmd/infra/aws/create.go#L91